### PR TITLE
Dresscode violation

### PR DESCRIPTION
**Pull request by nayana2020**
Dresscode violation functioanlities-add,view ,delete

@abramachandra @shekharsr7  Please review and approve

### CODE REVIEW

File: frontend/src/Dresscode.js [hunk 1]
```diff
@@ -1,12 +1,106 @@
...
+    }
+
+
+    getData = () => {
+        axios.get('/getDresscode')
+            .then(res => {
+                this.setState({people: res.data});
+                // call getData() again in 5 seconds
+                this.intervalID = setTimeout(this.getData.bind(this), 1000);
+            });
```
Comment (shekharsr7) [inline frontend/src/Dresscode.js:27]: wont need this call at every 1 sec interval

File: frontend/src/Dresscode.js [hunk 2]
```diff
@@ -1,12 +1,106 @@
...
+    addPerson(id, name,violation) {
+        const bodyFormData = {
+            name: name,
+            employeeId: id,
+            violation:violation
+        };
+        axios.post('/addDresscode', bodyFormData)
+            .then(res => {
+                this.setState({Dresscode: res.data});
+                console.log(this.state.Dresscode);
```
Comment (shekharsr7) [inline frontend/src/Dresscode.js]: remove console.log

File: frontend/src/Dresscode.js [hunk 3]
```diff
@@ -1,12 +1,106 @@
...
+
+    deletePerson(employeeId,checkinTime) {
+        const bodyFormData = {
+                   employeeId: employeeId,
+                   checkinTime:checkinTime
+               };
+               axios.post('/deleteDresscode', bodyFormData)
+                   .then(res => {
+                       this.setState({Dresscode: res.data});
+                       console.log(this.state.Dresscode);
```
Comment (shekharsr7) [inline frontend/src/Dresscode.js]: remove console.log

File: frontend/src/Dresscode.js [hunk 4]
```diff
@@ -1,12 +1,106 @@
...
+                   })
+    }
+
     render() {
+       // return (
+          // <section className="smart-filter-content">
+            //    <h1>Dresscode violation</h1>
+            //</section>
+        //);
```
Comment (shekharsr7) [inline frontend/src/Dresscode.js]: remove commented code

**Review by abramachandra — APPROVED** [review]

**Pull request merged**